tests/ui/specs/SliderButton/SliderButtonPage.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const {Page} = require('@enact/ui-test-utils/utils');
+
+class SliderButtonInterface {
+	constructor (id) {
+		this.id = id;
+		this.selector = `#${this.id}`;
+	}
+
+	get self () {
+		return $(this.selector);
+	}
+	knobValue () {
+		return $(this.selector + ' .SliderButton_SliderButton_knob');
+	}
+	clickableItem (index) {
+		return $(this.selector + ` .SliderButton_SliderButton_track>div:nth-child(${index})`);
```

Comment:
How about the below code for finding clickable items? I think it is more clear but you have to change then index number overall.
```suggestion
	const parent = $(this.selector + ' .SliderButton_SliderButton_track');
        return = parent.$$('.SliderButton_SliderButton_client')[index];
```